src/screens/TeamFormation/TeamFormationTabs.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import { createMaterialTopTabNavigator } from "@react-navigation/material-top-tabs";
+import { StyleSheet, View, Text, Pressable, Dimensions } from "react-native";
+import theme from '../../styles/theme.style.js';
+import Participants from './Tabs/Participants.js';
+import Teams from './Tabs/Teams';
+
+const Tab = createMaterialTopTabNavigator();
+const totalWidth = Dimensions.get('window').width;
+
+class TeamFormationTabs extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+      return (
+        <Tab.Navigator
+          screenOptions={{
+            tabBarLabelStyle: {
+              fontSize: theme.FONT_SIZE_SLIGHT_MEDIUM,
+              fontFamily: theme.FONT_SEMIBOLD,
+              letterSpacing: theme.LETTER_SPACING_SMALL,
+            },
+            tabBarStyle: {
+              backgroundColor: "transparent",
+              marginBottom: theme.SPACING_SMALL,
```

Comment:
you can also add some marginTop so that the tabs aren't too high up on the screen. I see them like this now
![IMG_6322DC7B39DA-1](https://user-images.githubusercontent.com/72771523/159760047-8754872d-6406-467d-ba69-f9d9cdbe9ae1.jpg)

Reply:
This is due to the bug with the tabs.  Once the screen container will be added, they will automatically be adjusted in a good position in the screen. Should I add some marginTop for now for more convenience to switch between tabs ? (you can slide the screen to switch between the tabs not click on the unreachable tab title)

Reply:
Ok cool then we can keep it like this for now